src/main/views/crown-warned-list.njk

```diff
@@ -0,0 +1,214 @@
+      {% endfor %}
+      {% if address.town | length %}
+        {{ address.town }}<br/>
+      {% endif %}
+      {% if address.county | length %}
+        {{ address.county }}<br/>
+      {% endif %}
+      {{ address.postCode }}
+    </p>
+
+    <div class="govuk-body list-info">
+      <p>{{ headingP1 }} {{ contentDate }}</p>
+      <p>{{ headingP2 }}</p>
+      <p>{{ headingP3 }}</p>
+      <p>{{ headingP4 }}</p>
+    </div>
+    {% if bill %}
+    <div class="govuk-body list-info-welsh">
+      <p>{{ headingP1Cy }} {{ contentDate }}</p>
+      <p>{{ headingP2Cy }}</p>
+      <p>{{ headingP3Cy }}</p>
+      <p>{{ headingP4Cy }}</p>
+    </div>
+    {% endif %}
+
+    <div class="govuk-grid restriction-list-section">
+      <div class="govuk-body">
+        <h3>{{ restrictionInformationHeading }}</h3>
+        <p class="govuk-body">{{ restrictionInformationP1 }}</p>
+        <div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+          <strong class="govuk-warning-text__text">
+            <span class="govuk-warning-text__assistive">Warning</span>
+            {{ restrictionInformationBoldText }}
+          </strong>
+        </div>
+        <p class="govuk-body">{{ restrictionInformationP2 }}</p>
+        <p class="govuk-body">{{ restrictionInformationP3 }}</p>
+        <p class="govuk-body">{{ restrictionInformationP4 }}</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>{{ restrictionBulletPoint1 }}</li>
+          <li>{{ restrictionBulletPoint2 }}</li>
+        </ul>
+        <p class="govuk-body">{{ restrictionInformationP5 }} <a href="#" class="govuk-link">{{ restrictionLink }}</a> </p>
+      </div>
+      {% if bill %}
+        <div class="govuk-body">
+          <h3><br/>{{ restrictionInformationHeadingCy }}</h3>
+          <p class="govuk-body">{{ restrictionInformationP1Cy }}</p>
+          <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+              <span class="govuk-warning-text__assistive">Warning</span>
+              {{ restrictionInformationBoldTextCy }}
+            </strong>
+          </div>
+          <p class="govuk-body">{{ restrictionInformationP2Cy }}</p>
+          <p class="govuk-body">{{ restrictionInformationP3Cy }}</p>
+          <p class="govuk-body">{{ restrictionInformationP4Cy }}</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>{{ restrictionBulletPoint1Cy }}</li>
+            <li>{{ restrictionBulletPoint2Cy }}</li>
+          </ul>
+          <p class="govuk-body">{{ restrictionInformationP5Cy }} <a href="#" class="govuk-link">{{ restrictionLinkCy }}</a> </p>
+        </div>
+      {% endif %}
+    </div>
+
+    <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+      {% for heatingType, rows in listData %}
```

Comment:
It looks like if the only session in the list is the toBeAllocation hearing type, a blank accordion is given at the top of the page, and then the correct populated one further down